### PR TITLE
commit of all p1 components

### PR DESCRIPTION
**Pull request by mmanjuprasad**
text,list,navigation,modal,rating,snackbar,grid

### CODE REVIEW

**Comment by kgajriaAgarwal** [comment]
captr1
design getting distotrted .can use text overflow elipsis here

**Comment by kgajriaAgarwal** [comment]
1. You have not created card with badge you can create that.
image
2. improve the gaping between the buttons
3. improve the buttons ui
image

**Comment by kgajriaAgarwal** [comment]
Kindly prepare a proper documentation site

**Comment by kgajriaAgarwal** [comment]
image
here the icon is not proper

**Comment by kgajriaAgarwal** [comment]
image
no proper spacing here

File: styles.css [hunk 1]
```diff
@@ -397,4 +397,334 @@ input[type=text]:focus {
...
+.h6 {
+  font-weight: 500;
+  font-size: 1rem;
+}
+
+.grey-color{
+  color: grey
+}
+.blue-color{
+  color: #0004ff!important;
```
Comment (nikhiljugale007) [inline styles.css:483]: try to avoid !important

File: styles.css [hunk 2]
```diff
@@ -397,4 +397,334 @@ input[type=text]:focus {
...
+  padding: 0 20px 0 20px;
+  height: 64px;
+  justify-content: space-between;
+}
+.nav-container-startsec{
+  display: flex;
+  align-items: center;
+}
+.nav-container-startsec > i{
+  margin-right: 32px;
```
Comment (nikhiljugale007) [inline styles.css:637]: try to use rems instead of px.

File: styles.css [hunk 3]
```diff
@@ -397,4 +397,334 @@ input[type=text]:focus {
...
+
+
+/* navigation header 2  */
+
+.nav-header {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 0 8px;
+  background: rgb(89, 134, 100);
```
Comment (nikhiljugale007) [inline styles.css:659]: try to define color variable and then use where u need them.

**Review by nikhiljugale007 — COMMENTED** [review]
add hover styles on button.
In social media card padding is not consistent.

UI and Color consistency can be improved.

One suggestion is you should make smaller PR. Max 2 Components in one PR.

**Pull request merged**